patches/libgd-2.3.3/libgd-export-pkg-config-build-in-subdirs.patch

```diff
@@ -0,0 +1,2623 @@
+
+
+
+
+
+
+
+<!DOCTYPE html>
```

Comment:
This patch file looks like it got the HTML rather than the raw patch. same for one or two others.

Reply:
Have rebased locally and force pushed.  The files should be good now.